Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.14.6 AS base
+FROM golang:1.15.8 AS base
```

Comment:
Yup, I do NOT see anything that looks like it'd give us any trouble: https://golang.org/doc/go1.15